packages/material-ui-lab/src/Autocomplete/Autocomplete.js

```diff
@@ -270,7 +270,7 @@ const Autocomplete = React.forwardRef(function Autocomplete(props, ref) {
     fullWidth = false,
     getLimitTagsText = (more) => `+${more}`,
     getOptionDisabled,
-    getOptionLabel = (x) => x,
+    getOptionLabel = (x) => x.label ?? x,
```

Comment:
```suggestion
    getOptionLabel = (option) => option.label ?? x,
```
to use a descriptive name similar to useAutocomplete.